corehq/apps/app_manager/models.py

```diff
@@ -1115,6 +1136,45 @@ def get_module(self):
     def get_case_type(self):
         return self._parent.case_type
 
+    def _add_save_to_case_questions(self, form_questions, app_case_meta):
+        def _make_save_to_case_question(path):
+            from corehq.apps.reports.formdetails.readable import FormQuestionResponse
+            # todo: this is a hack - just make an approximate save-to-case looking question
```

Comment:
eventually we should support save to case questions as a first class thing that `get_questions` knows how to parse and work with, but this seemed unneccessary to get this working. if we want those to ever be supported in form exports or form summaries we would need to do that work and then update this to use the real question.